tools/bin/ci_integration_test.sh

```diff
@@ -11,8 +13,14 @@ if [[ "$connector" == "all" ]] ; then
   echo "Running: ./gradlew --no-daemon --scan integrationTest"
   ./gradlew --no-daemon --scan integrationTest
 else
-  selected_integration_test=$(echo "$all_integration_tests" | grep "^$connector$" || echo "")
-  integrationTestCommand=":airbyte-integrations:connectors:$connector:integrationTest"
+  if [[ "$connector" == *"connectors"* ]] || [[ "$connector" == *"bases"* ]]; then
```

Comment:
why do we need this conditional? can't we just say you should always do it one way? 

if i'm interpreting right it seems like you can do: `connectors/source-postgres` or `source-postgres`? i think it's the `else` that feels a little iffy, because i think it'll work for `source-postgres` but it won't work for `base-normalization` since `:connectors:`is hard coded.

Reply:
This condition makes it so that it keeps both the old behavior and the new one.

You are right, the old behavior (which is `source-postgres`) won't work with `base-normalization` as you pointed out

However, I don't understand yet why the new behavior gets rejected when used with the `ref=XXX` arguments:
https://github.com/airbytehq/airbyte/pull/2531#issuecomment-804748708

but the old behavior does pass validation: https://github.com/airbytehq/airbyte/pull/2531#issuecomment-804383934

So since the new behavior doesn't fully replace the old one yet in terms of functionalities, I am leaving both options available...